src/constants.js

```diff
@@ -2,4 +2,4 @@ export const DOMAIN = 'accessible-colors.com';
 export const SITE_URL = `http://${DOMAIN}`;
 export const REPO = 'moroshko/accessible-colors';
 export const MIN_FONT_SIZE = 10;
-export const MAX_FONT_SIZE = 21;
+export const MAX_FONT_SIZE = 28;
```

Comment:
Now, 22pt and above are not reflected in the UI, as it's bounded by 28px.
Shall we change `MAX_FONT_SIZE` to 36?

Also, could you change the names `MIN_FONT_SIZE` and `MAX_FONT_SIZE` to be `MIN_FONT_SIZE_IN_PT` and `MAX_FONT_SIZE_IN_PT` (or `IN_PX`) to be more specific?